### ISSUE TITLE
Dysorthography

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10155]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
**Dysorthography** is a disorder of spelling which accompanies dyslexia by a direct consequence of the phonological disorder. In the American classification from the American Psychiatric Association (APA) and the classification from the World Health Organization (WHO), it is a subtype of specific learning disorder with impairment in written expression.

American Psychiatric Association - 5th edition (2013). Diagnostic and statistical manual for mental disorders. doi: 10.1176/appi.books.9780890425596. ISBN 978-0-89042-559-6

World Health Organization. "ICD-11 for Mortality and Morbidity Statistics"

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0010522

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:200 Isolated corpus callosum agenesis

**Your nano-attribution (ORCID)**
https://orcid.org/0009-0005-6714-5727

**Comment by pnrobinson** [comment]
I wonder if this is truly distinct from [Dysgraphia HP:0010526](https://hpo.jax.org/app/browse/term/HP:0010526)
see also https://www.ncbi.nlm.nih.gov/books/NBK559301/ ?

**Comment by pnrobinson** [comment]
@carolina-f  -- could you comment if there is a distinction between Dysgraphia and Dysorthography. Also, the latter word seems slightly mangled (-phy instead of -phia). Also the fact that it is listed as a disease/disorder means the above reference is not directly a phenotypic feature.

**Comment by carolina-f** [comment]
Dysgraphia is a writing disorder that affects a person's ability to write legibly. It may manifest as messy or illegible handwriting, issues with spacing and alignment, and difficulties in fine motor coordination required for writing and can have various causes, ranging from neuromuscular problems to difficulties in planning and motor coordination.
Instead, Dysorthography refers to a specific disorder in spelling, meaning the correct use of spelling rules and the accurate writing of words.Individuals with dysorthography may struggle to remember the correct spelling of words, make frequent spelling errors, and have difficulties applying grammatical and spelling rules.
Causes also can be diverse, including issues with visual memory, difficulties in language processing, or dysfunctions in brain areas associated with spelling.
But after looking DSM-5 I think both aren't explicitly classified as a separate disorder, they are considered category of "Specific Learning Disorder"

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000915
- label: Dysorthography
- parents: [Specific learning disability (HP:0001328)](https://hpo.jax.org/app/browse/term/HP:0001328)
- pmid: PMID:21348173
- def: A specific disorder of spelling. A score on a standardized spelling test is at least two standard errors of prediction below the level expected on the basis of the child's chronological age and general intelligence in the context of scores on reading accuracy and comprehension and on arithmetic that are within the normal range (within two standard deviations from the mean).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10155
- synonyms:
